app/models/message.rb

```diff
@@ -32,6 +32,7 @@ class Message < ApplicationRecord
 
   validates :sender_id, presence: true
   validates :type, :origin, length: { maximum: 255 }
+  validates :subject, presence: true
```

Comment:
A bit scary to add such a validation, when we know that there might be many messages without subject in the DB.
The good thing is that it seems that once created the Message objects are not modified, so maybe this is fine.

Reply:
yes but as per the discussion we need to make it mandatory and somehow it makes sense because we should never send empty messages some validation is must

Reply:
It makes sense to enforce good etiquette I guess.

Reply:
umm I think the requirement is to make subject required field and yes may be I can cofirm if character limit needed

Reply:
I have received comment from team on JIRA

[Nidhi Soni](https://issues.redhat.com/secure/ViewProfile.jspa?name=nsoni%40redhat.com) I think it is not necessary to add a minimal character limit. (Everything  message > 0 would be fine)

So I am resolving this conversation as we have already made subject required and other than that works fine for them.